### PR TITLE
Add command to build hybris-boot zip

### PR DESCRIPTION
**Pull request by DeadSquirrel01**
wee, I've created a script to build hybris-boot flashable zip file.
After . build/envsetup.sh and lunch devicename, the user can run:
$ boot_zip_gen
to build a flashable hybris-boot.zip in out/target/product/codename
The script is fully commented, so, I think I don't need particular explanations :D 
The script works with fstab V2 and V1 with mmcblk and bootdevice partitions. I tested on my fstab, hammerhead, bullhead and cheesburger fstab.

The zip can be used to make a rom with multirom or flashed as zip.
basically the scripts gets the name of boot partition from fstab and then create updater-script. Then it copies prebuilt update-binary and hybris-boot and creates the zip

### CODE REVIEW

**Comment by z3ntu** [comment]
I'm pretty sure we can build the update binary from source instead of using the binary you provided

**Comment by DeadSquirrel01** [comment]
I took it from multirom sources. Probably it can be built. There are some python files in releasetools. but I don't know python language. Anyway it works, you don't need a specific update-binary

**Comment by bhush9** [comment]
This pull request is closed due to violation of code of conduct.
